src/tests/unit/cpu/nodes/reorder_node_test.cpp

```diff
@@ -1,266 +1,430 @@
 // Copyright (C) 2018-2022 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-
+#include <common/blocked_desc_creator.h>
+#include <cpu_types.h>
+#include <edge.h>
 #include <gtest/gtest.h>
 #include <ie_common.h>
-
-#include <nodes/reorder.h>
-#include "nodes/input.h"
-#include <edge.h>
+#include <memory_desc/cpu_memory_desc_utils.h>
+#include <memory_desc/dnnl_memory_desc.h>
 #include <node.h>
+#include <nodes/reorder.h>
+
+#include <common/memory_desc_wrapper.hpp>
+#include <dnnl.hpp>
+#include <typeinfo>
```

Comment:
I'm not sure this include is still needed. Could you please check the other includes on whether they are still necessary?